kcidb/monitor/test_output.py

```diff
@@ -74,4 +74,15 @@ def test_min(self):
         self.assertEqual(message['X-KCIDB-Notification-ID'], notification.id)
         self.assertEqual(message['X-KCIDB-Notification-Message-ID'], "id")
         self.assertIn("Revision detected: ", message['Subject'])
-        self.assertIn("We detected a new revision!", message.get_payload())
+        text, html = message.get_payload()
+        self.assertEqual('text/plain', text.get_content_type())
+        self.assertEqual('utf-8', text.get_content_charset())
+        content = text.get_payload()
+        if text['content-transfer-encoding']:
```

Comment:
We don't really need to check if that header is there, it's pretty much guaranteed. Could you remove these checks?